docs/api_extra.rst

```diff
@@ -549,6 +549,12 @@ section <ndarrays>`.
 
 .. cpp:class:: template <typename... Args> ndarray
 
+   .. cpp:var:: is_ro
```

Comment:
Can you rename this ``ReadOnly``? The other static properties (e.g. `Scalar`) also have this upper case convention. I would expect `is_ro` to be a method.